dev/io.openliberty.jakarta.jsonb.compatibility/resources/io/openliberty/jakarta/jsonb/compatibility/resources/JsonbMessages.nlsprops

```diff
@@ -0,0 +1,35 @@
+###############################################################################
+# Copyright (c) 2022 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+# # {0} description of each insert field
+# MSG_DESCRIPTIVE_NAME_CWSJX0000=CWSJX0000I: This is a message with inserts {0}
+# MSG_DESCRIPTIVE_NAME_CWSJX0000.explanation=Explanation text for the message
+# MSG_DESCRIPTIVE_NAME_CWSJX0000.useraction=User action text for the message
+#
+#CMVCPATHNAME io.openliberty.jakarta.jsonb.compatibility/resources/io/openliberty/jakarta/jsonb/compatibility/resources/JsonbMessages.nlsprops
+#COMPONENTPREFIX JSON
+#COMPONENTNAMEFOR JSON Binding and Processing
+#ISMESSAGEFILE TRUE
+#NLS_ENCODING=UNICODE
+#
+#NLS_MESSAGEFORMAT_VAR
+#
+#   Strings in this file which contain replacement variables are processed by the MessageFormat 
+#   class (single quote must be coded as 2 consecutive single quotes ''). Strings in this file 
+#   which do NOT contain replacement variables are NOT processed by the MessageFormat class 
+#   (single quote must be coded as one single quote '). 
+# -------------------------------------------------------------------------------------------------
+
+# JSONB Messages 0001-4999
+# JSONP Messages 5000-9999
+
+JSON0001.feature.compatibility=JSON0001I: jsonb and jsonbContainer features are both enabled. The jsonbContainer feature supersedes the jsonb feature.
+JSON0001.feature.compatibility.explanation=The jsonbContainer feature allows users to provide their own implementation of the json binding API.  Whereas, the jsonb feature will provide a default implementation.
+JSON0001.feature.compatibility.useraction=The jsonb feature will not provide a json binding implementation, ensure that you have configured a BELL in your server configuration that implements the json binding API.
```

Comment:
The jsonb feature will not provide a json binding implementation, ensure that you have configured a BELL in your server configuration that implements the json binding API.

change to:

Ensure that you have configured a BELL in your server configuration that implements the Jakarta JSON Binding API.